x/ccv/provider/keeper/distribution.go

```diff
@@ -361,3 +361,29 @@ func (k Keeper) HandleSetConsumerCommissionRate(ctx sdk.Context, consumerId stri
 		commissionRate,
 	)
 }
+
+// TODO: this method needs to be tested
+func (k Keeper) ChangeRewardDenoms(ctx sdk.Context, denomsToAdd, denomsToRemove []string) []sdk.Attribute {
+	eventAttributes := []sdk.Attribute{}
+	for _, denomToAdd := range denomsToAdd {
+		// Log error and move on if one of the denoms is already registered
+		if k.ConsumerRewardDenomExists(ctx, denomToAdd) {
+			ctx.Logger().Error("denom %s already registered", denomToAdd)
```

Comment:
I do not believe that format specifiers work with `ctx.Logger().Error()`. We probably need something like:
`.Error("denom already registered", "denom", denomToAdd)`.